prime-router/src/main/kotlin/fhirengine/utils/FHIRBundleHelpers.kt

```diff
@@ -116,6 +117,26 @@ fun Bundle.addProvenanceReference() {
     }
 }
 
+/**
+ * Return true if Bundle contains an ELR in the MessageHeader.
+ *
+ * @return true if has a MesssageHeader that contains an R01 or ORU_R01, otherwise false.
+ */
+fun Bundle.isElr(): Boolean {
```

Comment:
RS currently receives test orders and results. Instead of `isElr`, perhaps a method like `getMessageType` that returns an enum value like `LAB_TEST_RESULT` or `LAB_TEST_ORDER` or `UNKOWN` would be more reusable? Then, I would say let's stamp the message if we get back either `LAB_TEST_RESULT` or `UNKOWN` just to be safe?

Implementing detection for `LAB_TEST_ORDER` is optional at this point as there is no need to distinguish this as of yet.

Reply:
Added some code for this with only LAB_RESULT or UNKNOWN as options.  Future features will need to add additional values to the enum.